pages/apis/managing_api_tokens.md

```diff
@@ -60,6 +60,14 @@ Removing access from a token will send a notification email to the token's owner
 >📘
 > Removing access from a token does not delete the token, however token owner cannot re-add your organization to their token's scope.
 
+## Limiting API Access by IP address
+>🚧 Beta Feature
+> This feature is not yet available to all organizations.
```

Comment:
I'll follow up about this part in a DM

Reply:
Hey @jfhinchcliffe (@mbelton-buildkite)

Yeah, I was just trying to be consistent across the board with the communication style.

My concern primarily stems from the GraphQL implementation. 

Given that this mutation becomes public knowledge and discoverable via the GraphQL explorer or introspection queries. 

If it _looks_ to be functional, but isn't functional due to the feature flag then I think we have a problem. Especially when we're dealing with security related features like an Allowlist. 

However, I'm happy to defer to @mbelton-buildkite for best practices here.

Reply:
changed to 'Private Beta' and updated screenshot above 😁

Reply:
Removed 'Beta' wording.